documentation/api/io.strimzi.api.kafka.model.kafka.KafkaClusterSpec.adoc

```diff
@@ -117,32 +117,28 @@ Be aware that if the configured image is not compatible with Strimzi images, it
 [id='property-kafka-logging-{context}']
 = Logging
 
-Kafka has its own configurable loggers, which include the following:
+WARNING: Kafka 3.9 and earlier versions use log4j1 for logging.
+For log4j1-based configuration examples, refer to the link:{DocArchive}[Strimzi 0.45 documentation^]. 
 
-* `log4j.logger.kafka`
-* `log4j.logger.org.apache.kafka`
-* `log4j.logger.kafka.request.logger`
-* `log4j.logger.kafka.network.Processor`
-* `log4j.logger.kafka.server.KafkaApis`
-* `log4j.logger.kafka.network.RequestChannel$`
-* `log4j.logger.kafka.controller`
-* `log4j.logger.kafka.log.LogCleaner`
-* `log4j.logger.state.change.logger`
-* `log4j.logger.kafka.authorizer.logger`
+Kafka has its own configurable loggers, which include the following:
 
-Kafka uses the Apache `log4j` logger implementation.
+* `rootLogger`
+* `kafka`
+* `orgapachekafka`
+* `requestlogger`
+* `requestchannel`
+* `requestchannel`
+* `controller`
+* `logcleaner`
+* `statechange`
+* `authorizer`
```

Comment:
Similarly to https://github.com/strimzi/strimzi-kafka-operator/pull/11254#discussion_r2008888734, we should clarify which packages / classes do these loggers cover as it is not completely obvious from their names.